docs/whats-new/index.md

```diff
@@ -16,6 +16,12 @@ You can help improve Pelican. Visit the [Feedback Page](/feedback) to learn how
 
 We’re continually improving Pelican. The following changes are listed by the date we completed each change.
 
+## 2.2.4: January 17, 2025
+
+- Changed how the textured background is loaded for performance reasons. Read more about this at [Background Textures](/foundation/agency-theming/#background-texture).
```

Comment:
This should be listed more prominently as a breaking change.

Reply:
Added instruction for restoring the dim badges